### ISSUE TITLE
Grammarly Browser Extension Causes Loss of Focus When Using `textarea` Elements

### ISSUE THREAD

**Issue by philippfromme** [bpmn-io/bpmn-js-properties-panel#810]
The [Grammarly browser extension](https://www.grammarly.com/browser) works by mutating the DOM. Whenever the user focuses a `textarea` element, the extension mutates it, which causes applications that expect no DOM mutations by anyone else to rerender (cf. https://github.com/facebook/draft-js/issues/616). The result is a loss of focus.

![brave_AxSg3NvxPd](https://user-images.githubusercontent.com/7633572/202491862-2a5a847b-de0a-4c7e-bdcc-b10823a225e3.gif)

The Grammarly team knows about this issue (cf. https://www.grammarly.com/blog/engineering/making-grammarly-feel-native-on-every-website/) and is working on a solution. In the meantime, adding a `data-gramm="false"` attribute to `textarea` element prevents Grammarly from mutating it.

![brave_LZkgXSOU45](https://user-images.githubusercontent.com/7633572/202492549-99c959ec-ce19-4e40-89ce-698a926a67fa.gif)

I'd suggest we add this attribute along with a comment to the [textarea](https://github.com/bpmn-io/properties-panel/blob/main/src/components/entries/TextArea.js) component.

**Comment by HanselIdes** [comment]
+1 on this feature
I was advising an internal project when this problem came up on a team call. The result was that instead of using Camunda Modeler for business-IT collaboration, they returned to an Excel sheet.

**Comment by philippfromme** [comment]
> The result was that instead of using Camunda Modeler for business-IT collaboration, they returned to an Excel sheet.

Ouch. It's not even a bug of the modeler. 😮‍💨

**Comment by nikku** [comment]
Us trying to deliver fixes so crappy software "on the internet" does not break our tools :wink:.

![typing](https://user-images.githubusercontent.com/58601/203165884-72088bba-c27d-40da-a186-a964189b98c5.gif)

**Comment by HanselIdes** [comment]
I've never seen an autosave action triggered by work-in-progress text changes. Usually, I'd expect them to be triggered after de-focusing the active text box

**Comment by nikku** [comment]
> I've never seen an autosave action triggered by work-in-progress text changes. Usually, I'd expect them to be triggered after de-focusing the active text box

@HanselIdes Is this feedback related to this issue? If so, could you elaborate?

**Comment by HanselIdes** [comment]
Sure - I posted my issue in [Slack](https://camunda.slack.com/archives/C034N0E5YCX/p1668780396135319) with a video. Maybe I'm not used to seeing the autosave animation, but I found it distracting because it was firing _all the time_.

My expectation is that changing a field is an atomic action. I don't expect it to be saved halfway through editing the field, especially because it can be invalid (e.g. FEEL expression).
